### PR TITLE
Ashlesha  - Update Description for Microsoft Teams All Sample App

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/msgext-search-sso-config/csharp/AppManifest/manifest.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
```
Comment (Wajeed-msft) [inline samples/msgext-search-sso-config/csharp/AppManifest/manifest.json:2]: Why are we adding new manifest for this sample?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/msgext-search-sso-config/csharp/AppManifest/manifest.json (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/msgext-search-sso-config/csharp/AppManifest/manifest.json]: We have removed the new manifest, as it is not required.

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, approving!

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/11679507773

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**